packages/wdio-browserstack-service/src/types.ts

```diff
@@ -244,3 +257,8 @@ interface StepData {
 interface Failure {
     backtrace: string[]
 }
+
+export interface BrowserStack extends WebdriverIO.Browser  {
```

Comment:
See https://webdriver.io/docs/customcommands#extend-type-definitions how to extend the types so that users will see these commands too, checkout the devtools service, sauce service or shared store service how it is being done there.